docs/src/request.rst

```diff
@@ -25,6 +25,14 @@ Data types
 Public members
 ^^^^^^^^^^^^^^
 
+.. c:member:: uv_loop_t* uv_req_t.loop
+
+    Loop that started this request and where completion will be reported.
+    Readonly.
+    
+    .. versionadded:: 1.25.0
+       Pulled up from sub-types.
```

Comment:
I can add every thing, but I doubt more wording will express things more than that. I chose these words because of the refactoring pattern that happened, and should be sufficient.
But if there is a consensus about a better wording, I'll change that.

Reply:
Maybe this?

> Moved from derived types (`uv_connect_t`, `uv_fs_t`, etc.) to `uv_req_t`.